fleetconfig-controller/api/v1alpha1/fleetconfig_types.go

```diff
@@ -306,6 +307,21 @@ type Spoke struct {
 	// ClusterARN is the ARN of the spoke cluster.
 	// This field is optionally used for AWS IRSA registration authentication.
 	ClusterARN string `json:"clusterARN,omitempty"`
+
+	// AddOns are the add-ons to enable for the spoke cluster.
+	AddOns []AddOn `json:"addOns,omitempty"`
+}
+
+// AddOn enables add-on installation on the cluster.
+type AddOn struct {
+	// The name of the add-on being enabled. Must match one of the default or manually configured add-on names.
+	ConfigName string `json:"configName"`
+
+	// The spoke cluster namespace to install the add-on in. If left empty, installs into the "open-cluster-management-addon" namespace.
```

Comment:
this was missed now that the struct is not specific to spokes:


```suggestion
	// The namespace to install the add-on in. If left empty, installs into the "open-cluster-management-addon" namespace.
```